symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/activity/SlashCommandsWithArgs.java

```diff
@@ -0,0 +1,29 @@
+package com.symphony.bdk.examples.activity;
+
+import static com.symphony.bdk.core.config.BdkConfigLoader.loadFromSymphonyDir;
+
+import com.symphony.bdk.core.SymphonyBdk;
+import com.symphony.bdk.core.activity.command.SlashArgumentActivity;
+import com.symphony.bdk.core.activity.command.SlashMentionArgumentActivity;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class SlashCommandsWithArgs {
+
+  public static void main(String[] args) throws Exception {
+    final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));
+
+    bdk.activities().register(new SlashArgumentActivity("/echo {one} {two}", true, (context, slashArgs) ->
+        bdk.messages().send(context.getStreamId(), "Args: " + slashArgs)
+    ));
+
+    bdk.activities().register(new SlashMentionArgumentActivity("/mentions {one} {two}", true, (context, slashArgs) ->
```

Comment:
I'm really not in favour of having a different activity for mentions. I'd expect something like: 
```java
bdk.activities().register(new SlashCommandActivity("/echo {one} @{two} #{three} ${four}", true, (context, slashArgs) ->
        bdk.messages().send(context.getStreamId(), "Args: " + slashArgs)
)); 
```

Reply:
Issue here is that we would have two types of consumers:
- either Consumer<CommandContext> without arguments
- or BiConsumer<CommandContext, Map<String, Object>> with arguments
One way could be to accept both, put the arguments map in the context and call whatever handler is provided.

Reply:
Can we put the arguments inside the context object?